### DISCUSSION TITLE
Healthcheck URL

### DISCUSSION THREAD

**Discussion by brpaz** [uxbox/uxbox#2538] [Q&A]
Hello.
I am deploying Penpot in K8s and I want to configure health checks. I checked that a "health" endpoint was added in release 1.12, but I cannot find the correct URL to call it.

I tried "/health", "/healthz", but I always get a 404 error.

What is the correct route?

**Comment by niwinz** [comment]
Because it is not exposed on the nginx, is only exposed on direct access to the backend port. Please open an issue for discuss about this. Discussions are deprecated.
